template/.circleci/config.yml

```diff
@@ -0,0 +1,214 @@
+version: 2
+jobs:
+  build_and_test_features:
+    docker:
+      - image: circleci/node:lts
+    environment:
+      - PROJECT_NAME: "{{projectName}}"
```

Comment:
Should this be ```{{projectName}``` or ```{{serviceName}}```? I have introduced two new variables:

```serviceName = boo```
```capitalizedServiceName = Boo```

when ```projectName = micro-boo```

I think, here, it should still be ```{{projectName}}```, but please double check.